Slim/Schema.pm

```diff
@@ -1470,9 +1470,13 @@ sub _createComments {
 }
 
 sub _createWork {
-	my ($self, $work, $workSort, $composerID, $create) = @_;
+	my ($self, $work, $workSort, $composerID, $create, $track, $album) = @_;
 
 	if ( $work && $composerID ) {
+		if (ref $work eq 'ARRAY') {
+			$log->error("-- Found mutiple work tags for album=$album, track=$track. Setting Work=@$work[0]");
```

Comment:
I'd rather log the file's path (url) than album/track titles. That's what you'd need to fix the situation, after all. And should we really consider this an error? Or would a warning be good enough? Some people panic when they see an error...